packages/stacked_tools/test/services_tests/revert_template_service_test.dart

```diff
@@ -0,0 +1,855 @@
+import 'package:stacked_tools/src/locator.dart';
+import 'package:stacked_tools/src/services/revert_template_service.dart';
+import 'package:stacked_tools/src/templates/template_constants.dart';
+import 'package:test/test.dart';
+
+import '../helpers/test_helpers.dart';
+
+RevertTemplateService _getRevertService() => RevertTemplateService();
```

Comment:
Common practice in tests is to call this _getService() it will be generated with the tests in the future. If you're testing a service you can get your service using _getService() if you're testing a ViewModel you can get it using _getViewModel()